tests/exploits/cve_2024_47533_test.py

```diff
@@ -0,0 +1,200 @@
+"""Unit tests for Agent Asteroid: CVE-2024-47533"""
+
+import xmlrpc.client
+from unittest import mock
+
+import requests
+import requests_mock as req_mock
+
+from agent import definitions
+from agent.exploits import cve_2024_47533
+
+
+def create_mock_vulnerable_response() -> str:
+    """Create a mock response that simulates a successful authentication bypass."""
+    return """
+    <?xml version='1.0'?>
+    <methodResponse>
+        <params>
+            <param>
+                <value>
+                    <string>token123</string>
+                </value>
+            </param>
+        </params>
+    </methodResponse>
+    """
```

Comment:
Should be a const, or just use it directly.

Reply:
What's wrong with it this way?